site/content/docs/commands/deploy.en.md

```diff
@@ -5,33 +5,57 @@ $ copilot deploy
 
 ## What does it do?
 
-This command is used to run either [`copilot svc deploy`](../commands/svc-deploy.en.md) or [`copilot job deploy`](../commands/job-deploy.en.md) under the hood. The steps involved in `copilot deploy` are the same as those involved in `copilot svc deploy` and `copilot job deploy`:
+This command is used to run either [`copilot svc deploy`](../commands/svc-deploy.en.md) or [`copilot job deploy`](../commands/job-deploy.en.md) under the hood.
```

Comment:
i think we can just get rid of this and rely on the line below (maybe need to reword it a bit, but) - it's more complicated that this now